### PR TITLE
Update GUI to suit the theme of TrackBeau

### PR DESCRIPTION
**Pull request by flairekq**
- Changed colour scheme of the application
- Able to toggle between the view for Customers, Bookings and Statistics

### CODE REVIEW

File: src/main/java/seedu/trackbeau/ui/StatisticsPanel.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package seedu.trackbeau.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.trackbeau.model.customer.Customer;
+
+public class StatisticsPanel extends UiPart<Region> {
```
Comment (isabelteo) [inline src/main/java/seedu/trackbeau/ui/StatisticsPanel.java:8]: maybe can add documentation?

**Review by isabelteo — COMMENTED** [review]
I think it's good to go, though I am not sure how much documentation we need to do for the UI side

File: src/main/java/seedu/trackbeau/ui/StatisticsPanel.java (hunk 1 above)
Comment (flairekq) [reply src/main/java/seedu/trackbeau/ui/StatisticsPanel.java:8]: oo like?

File: src/main/java/seedu/trackbeau/ui/MainWindow.java [hunk 2]
```diff
@@ -66,6 +83,46 @@ public MainWindow(Stage primaryStage, Logic logic) {
...
+        this.labels.add(bookingsLabel);
+        this.labels.add(statisticsLabel);
+
+        customersLabel.getStyleClass().add("selected");
+        for (Label l : this.labels) {
+            setLabelOnMouseClickEvent(l);
+        }
+    }
+
+    private void setLabelOnMouseClickEvent(Label label) {
```
[the remark this answers is not in the document]
Comment (flairekq) [reply src/main/java/seedu/trackbeau/ui/MainWindow.java:98]: u mean the switch case part?

File: src/main/java/seedu/trackbeau/ui/StatisticsPanel.java (hunk 1 above)
Comment (hanhuiice) [reply src/main/java/seedu/trackbeau/ui/StatisticsPanel.java:8]: I think she meant java doc for the class.

**Pull request merged**